app/models/sample.rb

```diff
@@ -191,6 +191,7 @@ def results_folder_files
     if pr.pipeline_version.to_f >= 2.0
       file_list = list_outputs(pr.output_s3_path_with_version)
       file_list += list_outputs(sample_output_s3_path)
+      file_list += list_outputs(pr.postprocess_output_s3_path)
       file_list += list_outputs(pr.postprocess_output_s3_path + '/' + ASSEMBLY_DIR, 2)
```

Comment:
do we still need the "assembly" prefix if they're grouped together now? (display_prefix=2 could be removed)

Reply:
Agreed